uyuni/base/uyuni-base.spec

```diff
@@ -26,14 +26,20 @@
 %define apache_group www
 %else
 %define www_path %{_var}
+%if 0%{?rhel}
+%define apache_user root
+%define apache_group root
+%else
 %define apache_user apache
 %define apache_group apache
 %endif
+%endif
 
 Name:           uyuni-base
 Version:        4.2.1
 Release:        1
 Url:            https://github.com/uyuni-project/uyuni
+Source:		noop.tar.gz
```

Comment:
Is this empty tar really needed to build on RHEL?
(just try to learn something new)

Reply:
Actually no. It is necessary for the Copr build process (not sure how OBS takes it on CentOS packages). Personally, I would like to keep it in to have that flexibility. Otherwise I can take it out.

Reply:
@juliogonzalez any objections?

Reply:
Not from me, but will autobuild accept it?

Reply:
I think yes, but what about tito? 
What about adding a README and a LICENSE file and make it a standard package with tar like all the other?

Reply:
Ah, right, tito would rewrite this.

I'd say it's better if we convert this to a standard package, yes.

Reply:
Can you please guide me what to do? What is a standard package?

Reply:
- copy the LICENSE file from uyuni/common-libs/ to uyuni/base/
- modify uyuni-base.spec
  * add `Source0:        %{name}-%{version}.tar.gz`
  * add `%license LICENSE` (to filelist `%files common`)

I think thats all.

Reply:
Done.